utils/defillama.ts

```diff
@@ -1,68 +1,36 @@
 import axios from "axios";
 
-const baseUrl = "https://api.llama.fi/protocol/uma-protocol";
-const defillamaUrls: string[] = [
-  "https://api.llama.fi/protocol/polymarket",
-  "https://api.llama.fi/protocol/sherlock",
-  "https://api.llama.fi/protocol/jarvis-network",
-  "https://api.llama.fi/protocol/across",
-  "https://api.llama.fi/protocol/uma-protocol",
-];
+const baseUrl = "https://api.llama.fi/oracles";
 
 interface TvlData {
   date: string;
   totalLiquidityUSD: number;
 }
 
-export async function getTvl(url: string = baseUrl) {
+export async function getDefillamaTvl(url: string = baseUrl) {
   const { data } = await axios.get(url);
-  const defillamaTvlData: TvlData[] = data.tvl.slice(-1);
-  const latestTvl = defillamaTvlData[0].totalLiquidityUSD;
-  return latestTvl;
-}
 
-export async function getPercentChange(url: string = baseUrl) {
-  const { data } = await axios.get(url);
-  const defillamaTvlData: TvlData[] = data.tvl.slice(-2);
-  const latestTvl = defillamaTvlData[1].totalLiquidityUSD;
-  const previousDayTvl = defillamaTvlData[0].totalLiquidityUSD;
-  return {
-    latestTvl,
-    previousDayTvl,
-  };
-}
+  const result = Object.keys(data.chart).map((key) => [key, data.chart[key]]);
```

Comment:
It seems like data.chart is already an array, and this step isnt necessary, you can just slice data.chart to get the last element directly?

If data.chart is an object its a differen story and this might be a potential bug since js does not gauarntee ordering in an object